docker/Dockerfile-alpine

```diff
@@ -0,0 +1,14 @@
+FROM python:3.7.3-alpine
+LABEL owner="MajorcaDevs"
+COPY ./mdbackup/ /mdbackup 
+COPY ./MANIFEST.in /mdbackup
+COPY ./README.md /mdbackup
+COPY ./setup.py /mdbackup
+WORKDIR mdbackup
+
+RUN mkdir /backups && apk update && apk upgrade && \
```

Comment:
Better not to update/upgrade the container. Increases the size of the real image, because of the differential layers.